src/Page/Submission/Team1/Algorithmold.js

```diff
@@ -0,0 +1,89 @@
+// Sherman Grewal
```

Comment:
I guess this file is not used. Probably delete this if possible. FYI: pushing unused code to master is not good practice.